Assets/Scripts/Enemies/Bosses/SpikeBall.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
+
+public class SpikeBall : EnemyMovement
+{
+
+    float timer = 0;
+
+    public override Vector3 PositionTransform(EnemyPosition position, float deltaTime)
+    {
+        return position.Up * deltaTime;
+    }
+
+    public override Vector3 RotationTransform(EnemyPosition position, float deltaTime)
+    {
+        return Vector3.zero;
+    }
+
+    public override void Update()
+    {
+        base.Update();
+
+        timer += Time.deltaTime;
+        if (timer>=3)
+        {
+            shootSpikes();
+            timer = -50;
```

Comment:
Is this so that it doesn't shoot again while on screen? A simple boolean could also achieve this.